base/src/test/resources/failure/syntax/bad-use-as.aya.txt

```diff
@@ -5,7 +5,21 @@ In file $FILE:6:20 ->
   6 │   open A using (True, False as True)
     │                       ╰───────────╯
 
-Warning: The name `True` introduces ambiguity and can only be accessed through a
-         qualified name
+Warning: The name `True` shadows a previous local definition from outer scope
```

Comment:
The "previous local definition" does not sound "local" to me 🤔

Reply:
Maybe we need a new `Problem`.